### DISCUSSION TITLE
Now that Discussions are here, can we have Issues restrictions?

### DISCUSSION THREAD

**Discussion by gilgongo** [community/community#3679] [Projects and Issues]
On a public repository, there is no way of stopping anyone with a GH account raising an Issue, This means Issues have traditionally been a mess of actionable work "tickets", open-ended chats about random stuff, and (particularly annoying for maintainers) items that look like actionable things, but are then revealed in the comments to have changed completely into something else. 

The existence of Discussions finally makes it possible to have an Issues list as an actionable list of things to do, manage those easily in Projects and have all the "fermenting" of issues done in Discussions. The main weapon in this fight is the fantastic "Convert to Discussion" feature.

**HOWEVER** - the Utopia of "Issues for action, Discussions as community" can't happen properly because random folk can still post random things in Issues, Nobody reads Issue templates.

When will Discussions finally come of age with admins able to restrict Issue posting to team members only?

**Comment by karussell** [comment]
:+1: 

It would be also important to limit access for pull requests e.g. allow only contributors to create them.

(btw: probably the Category of this question / feature request should be moved to "Issues" :) )

**Comment by UncleClapton** [comment]
With the introduction of YAML-based issue forms, having the option to restrict who can use specific forms would be a great move forward. I would like to have an open form for bug reporting while leaving other forms (such as task tracking issues) only available to collaborators, maintainers, or even only admins.

**Comment by Poopooracoocoo** [comment]
Also see https://github.com/dear-github/dear-github/issues/293 and https://github.com/orgs/community/discussions/40363

GitHub issues needs more controls. Many maintainers want to limit who can create issues and who can create discussions. A lot want only discussions to be the place for users to report issues or receive support. This would make actual development significantly easier and reduces the burden from having to triage issues. Many don't get paid to do it so it's just consuming time and causes significant stress. This is the worst aspect of open source. There are ways to work around it, but such things remove the "Hub" in "GitHub". This is the biggest problem with issues and discussions at the moment. Too many support requests and incomplete bug reports and crazy suggestions and not enough contributions. There really needs to be a clear separation between contributors and users in GitHub itself. I've specifically got the ExplorerPatcher repository in mind. I've seen the toll the burden of triaging issues took on the maintainer. I've also got the GDLauncher and Ruffle repositories in mind, as well as a handful of others. Microsoft may be able to afford to hire dedicated issue triagers, but other companies and FOSS orgs and individuals can't.

Many repos just use projects as a way to separate contributors (and other well-made) issues from the flood of other stuff. It'd be great to see them use projects in a more efficient way.
